frontend/simple/views/containers/contributions/monetary-method-form/Receiving.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <monetary-method-template
+    :title="L('Receiving Mincome')"
+    :methodActive="methodActive"
+    @method-change="updatePaymentMethod"
+    :submitText="L('Save changes')"
```

Comment:
@greg I have a question. In the Contributions page, will the user ever see a state where we doesn't receive any monetary contribution?

I ask that to know if the submit button will ever have a "Add receiving method" copy instead of "Save changes" or if `getManualAmount` will return null...

Reply:
I'm not entirely sure I understand your question, let's chat about this on Monday (tomorrow) then!

Reply:
@sandrina-p 

>  In the Contributions page, will the user ever see a state where we doesn't receive any monetary contribution?

There are two situations:

1. If they haven't set a receiving payment method
2. If they have set a receiving payment method, and they've limited the amount they receive from the group to $0/month

There are some significant UI changes that still need to be made for the receiving column though. For example, we need to handle the situation where they do not have a receiving payment set, and right now the way this PR does it isn't very good.

It would be better if it was clearer that no receiving payment method is set immediately, without having to click on anything. Similar to how you have dashed boxes on the right, and also how you asked for receiving payment methods in some of your mockups.

So there should maybe be a dashed box that says, "No receiving payment method set!", that when you click it, it gives options on how to receive the payment.

And then a separate UI for limiting the amount received, once set.

I also wrote some comments here (see the number (2) note): https://app.zeplin.io/project/5b3dbc2551fc18af1295e6e2/screen/5b6a58f6daf99015f1639679

Reply:
Thanks for the clarification, I wish I had made this question earlier, because it's a important missing step on user journey that should be defined before moving on UI implementation.

The good part is so far the UI implementation is flexible! Also it's awesome this is a separated PR from non-monetary contributions UX (that is already closed!). 

I'm realizing the way monetary contributions work are closely related to #483, so I suggest to move the UX discussion to there and come back to this PR once we better define the workflow. What do you think?

Reply:
That sounds like a good idea, so hold off on merging this for now?

Reply:
Yes, I've added the tag "WIP|Blocked" for now :)